File: TickerQ.EntityFrameworkCore/Entities/CronTickerEntity.cs
using TickerQ.EntityFrameworkCore.Entities.BaseEntity;

namespace TickerQ.EntityFrameworkCore.Entities
{
    public class CronTickerEntity : BaseTickerEntity
    {
        public virtual string Expression { get; set; }
        public virtual byte[] Request { get; set; }
        public int Retries { get; set; }
        public int[] RetryIntervals { get; set; }
    }
}

File: hub/sdks/node/src/models/CronTickerEntity.ts
import type { TickerRequestPayload } from './TimeTickerEntity';

export interface CronTickerEntity {
    id: string;
    function: string;
    description: string | null;
    initIdentifier: string | null;
    createdAt: string;
    updatedAt: string;
    expression: string;
    request: TickerRequestPayload;
    retries: number;
    retryIntervals: number[] | null;
    isEnabled: boolean;
}
